app/main.py

```diff
@@ -1 +1,32 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self,
```

Comment:
After opening the parentheses, write all the arguments on a new line, including the first one,